tests/services/copy_project/test_copy_project.py

```diff
@@ -17,3 +20,45 @@ def test__generate_project_title(existing_projects, expected, mocker):
     copy_proj = CopyProject(kili.auth)
     mocker.patch.object(copy_proj, "projects", return_value=existing_projects)
     assert copy_proj._generate_project_title("Title") == expected
+
+
+def test__labels_sort_1():
+    json_response_array = [1, 2, 3]
+    author_id_array = [4, 5, 6]
+    seconds_to_label_array = [7, 8, 9]
+    model_name_array = [None, "yolo", None]
+    label_type_array = ["DEFAULT", "AUTOSAVE", "AUTOSAVE"]
```

Comment:
model_name are only for PREDICITION labelType. All other labelType must have model_name to None